test/siblings_test.exs

```diff
@@ -107,6 +107,26 @@ defmodule Siblings.Test.Siblings do
     refute Process.whereis(MySiblingsWithKiller)
   end
 
+  test "Worker-FSM with Throttler" do
+    Siblings.start_child(Siblings.Test.WorkerFSM, "MyWorkerFSM", %{pid: self()},
+      name: MySiblingsWithKiller,
+      interval: 100,
+      throttler: [:s2]
```

Comment:
⚠️  this is the trick: we tell the process that `perform/3` in `:s2` state has to be throttled.